document/static/js/modules/editor/navigator/model.js

```diff
@@ -0,0 +1,28 @@
+export const navigatorFilterModel = ()=> ({
+    content: [
+        {
+            title: 'Heading 1',
```

Comment:
all text that we show to end users has to be inside `gettext()` so that it gets translated when showing the interface in other languages. So this should be `gettext('Heading 1')`